src/main/java/com/bandwidth/iris/sdk/model/BaseModel.java

```diff
@@ -1,19 +0,0 @@
-package com.bandwidth.iris.sdk.model;
-
-import com.bandwidth.iris.sdk.IrisClient;
-
-/**
- * Created by sbarstow on 11/11/14.
- */
-public abstract class BaseModel {
```

Comment:
Is `BaseModel` really necessary? There are only a few classes that extend this but don't actually use the `client`. The client is always passed into some static method on the model class. The "model" itself doesn't really need to know anything about `IrisClient`.